code/modules/species/species_bodytype.dm

```diff
@@ -422,3 +448,9 @@ var/global/list/bodytypes_by_category = list()
 		var/organ_type = replacing_organs[organ_tag]
 		var/obj/item/organ/internal/new_innard = new organ_type(limb.owner, null, limb.owner.dna, src)
 		limb.owner.add_organ(new_innard, GET_EXTERNAL_ORGAN(limb.owner, new_innard.parent_organ), FALSE, FALSE)
+
+/decl/bodytype/proc/get_typical_user_species()
```

Comment:
If this is only used for validation, I'd like that to be specified—something like `get_validation_user_species()` instead.